job-server/config/dse.conf

```diff
@@ -0,0 +1,58 @@
+# Template for a Spark Job Server configuration file
+# When deployed these settings are loaded when job server starts
+#
+# Spark Cluster / Job Server configuration
+spark {
+  # Spark Master will be automatically learned via the DSE
+  # spark.master will be passed to each job's JobContext
+  # master = "local[4]"
+  # master = "mesos://vm28-hulk-pub:5050"
+  # master = "yarn-client"
+
+  # Default # of CPUs for jobs to use for Spark standalone cluster
+  job-number-cpus = 4
+
+  jobserver {
+    port = 8090
+    jar-store-rootdir = /tmp/jobserver/jars
```

Comment:
This is a spark jobserver default, i'm not sure where we would like to put them otherwise.